routes/items.js

```diff
@@ -35,4 +35,15 @@ router.post( '/:id/completed', ( request, response ) => {
     .catch( error => response.json({ success: false, id, message: error.message }))
 })
 
+router.post( '/:id/edit', ( request, response ) => {
+  const Item = request.app.get( 'models' ).Item
+
+  const { id } = request.params
+  const { title, description } = request.body
```

Comment:
I think we're missing an opportunity to generalize this endpoint (and potentially get rid of the `/completed` endpoint, above):
```js
router.post( '/:id'. ( request, response ) => {
  const Item = request.app.get( 'models' ).Item

  const { id } = request.params

  Item.update( request.body, { where: { id, user_id: request.user.id })
    // etc.
})
```
When I stepped back and looked at the two `/:id/something` methods, they were almost exactly the same - varying only in the data that is read from request.body, then forwarded to the Item for update.